### ISSUE TITLE
(Classic Raid) Zul Gurub Bug List (Vanilla Guild)

### ISSUE THREAD

**Issue by chaostheory102** [dalaranwow/dalaran-wow#4124]
[Zul Gurub Bug.docx](https://github.com/dalaranwow/dalaran-wow/files/686460/Zul.Gurub.Bug.docx)
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by AgeAgainst** [comment]
Just a note for you -

Your video for High Priest Arlokk encounter is a video for Onyxia, not Arlokk.  The proper video is:

**Comment by Rushor** [comment]
i will post everything in there for now:

Zul Gurub Bug
High Priest Jeklik – 3 Problems
Problem 1
	Current Behavior: 
		Cannot Be pulled down without a DK
	Expected Behavior:
		Should come down due to proximity range or taunt
	Steps To Reproduce
1. Clear mobs in boss area
2. Get close to boss, unable to pull without DK death grip
	Proof
	https://www.youtube.com/watch?v=T9_oghoERTc view how boss is pulled
Problem 2
	Current Behavior
		The bats that are spawned cannot be seen
	Expected Behavior
		The bats that are spawned should be visible
	Steps To Reproduce
1. Engage High Priestest Jeklik
2. DPS until bats are spawned
3. View as they are not visible
	Proof
	https://www.youtube.com/watch?v=T9_oghoERTc Bats are visible
Problem 3
	Current Behavior
		Spawned Bats cannot be targeted
	Expected Behavior
		Spawned Bats should be targetable
	Steps To Reproduce
1. Engage High Priestest Jeklik
2. DPS until bats are spawned
3. View as they are not targetable
	Proof
https://www.youtube.com/watch?v=T9_oghoERTc – View nameplates and how objects with nameplates are targetable
Bloodlord Mandokir – 5 Problems
	Problem 1
		Current Behavior
			The ghosts in the boss area do not respawn individuals who have died
		Expected Behavior
			When someone dies a ghost will resurrect a dead raid member at random
		Steps to Reproduce
1. Engage Bloodlord Mandokir
2. Die
3. Wait and not get a respawn
		Proof
		http://wowwiki.wikia.com/wiki/Bloodlord_Mandokir_(original) – 1st paragraph
	Problem 2
		Current Behavior
			Bloodlord Mandokir can be glitched on the stairs
		Expected Behavior
			Bloodlord Mandokir should not be able to be bugged on the stairs
		Steps to Reproduce
1. Do not engage Bloodlord Mandokir by killing speaker
2. Pull Speaker run up stairs and jump on left platform to the stairs
3. Run to furthermost corner and range dps speaker., then boss and raptor.
		SCREENSHOT NEEDED
	Problem 3
		Current Behavior
			Does not reset properly after a raid wipe. He runs away and disappears
		
Expected Behavior
			Should reset after a wipe to allow for another attempt	
		Steps to Reproduce
1. Engage Bloodlord Mandokir
2. Entire raid should die/wipe,
3. Observe as he dissapears
		SCREENSHOT NEEDED
	Problem 4
		Current Behavior
			Does the level up animation, but does not grow in size
		Expected Behavior
			When he kills someone should level up, do more damage, and grow in size
		Steps to Reproduce
1. Engage Bloodlord Mandokir
2. Members of the raid should die
3. Observe his “level up” animation presents, size increase is not. Damage sometimes does not increase
		Proof
			wowwiki.wikia.com/wiki/Bloodlord_Mandokir_(original) 
	Problem 5
		Current Behavior
			Focus Gaze is not LOS’able(line of sight)
		Expected Behavior
			Should be able to Line of Sight his gaze/incoming charge
		Steps to Reproduce
1. Engage Bloodlord
2. Wait for Gaze and attempt to LOS
		Proof
			wowwiki.wikia.com/wiki/Bloodlord_Mandokir_(original) 
High Priest Thekal – 2 Problems
	Problem 1
		Current Behavior
			Thekal and his adds can be killed separately without resurrecting each other
		Expected Behavior
Thekal and his adds have to be killed within 10 seconds of one another, if not they resurrect each other
		Steps to Reproduce
1. Engage High Priest Thekal
2. Kill Thekal or one of the adds, view as they don’t resurrect each other
		Proof
			http://wowwiki.wikia.com/wiki/High_Priest_Thekal 
Problem 2
		Current Behavior
			Thekal does not respawn as a tiger upon his 1st death
		Expected Behavior
			After he dies for the 1st time, thecal should respawn as a Tiger
		Steps to Reproduce
1. Engage and kill High Priest THekal
2. View as he does not respawn as a tiger, or at all
		Proof
			http://wowwiki.wikia.com/wiki/High_Priest_Thekal 
High Priest Arlokk – 2 Problems
	Problem 1
		Current Behavior
		High Priest Arlokk and her adds do not do as much damage as they should
		Expected Behavior
		Arlokk, should almost 1 shot people on her opener, she does not. Her adds should do more damage and head towards healers or caster, they do not.
		Steps to Reproduce
1. Begin boss fight. Observe
Proof
https://www.youtube.com/watch?v=OsCi-1oGYso 
	



Problem 2
		Current Behavior
		Can Summon more than  1 boss from Gong
Expected Behavior
		Should only be able to summon 1 boss
		Steps to Reproduce
1. Click gong to summon boss in quick succession
		Proof
Jindo the Hexxer – 3 Problems
	Problem 1
		Current Behavior
			Brain wash totem he spawns does not Mind Control Raid Members
		Expected Behavior
			Brain Wash totem should mind control nearest raid member
		Steps to Reproduce
1. Begin boss fight
2. Let Jindo spawn a Brain Wash totem, and not get mind controlled
		Proof
			http://wowwiki.wikia.com/wiki/Jin%27do_the_Hexxer - Abilities
	Problem 2
		Current Behavior
			Jindo’s Shades do not interrupt spellcasting when they attack
		Expected Behavior
			Should Interrupt Spellcasting like normal mobs
		Steps to Reproduce
1. Engage Jindo, let him spawn an add
2. Cast and view as there is no reprocussion for being hit by an add
	Problem 3
		General problem
			Jindo’s shades do not do as much damage as they should
Hakkar the Soulflayer
		Current Behavior
			Hakkar does not do Blood Siphon or Corrupted Blood
		Expected Behavior
			Hakkar should blood siphon and Corrupted Blood
		Blood Siphon: Every 90 seconds from the time that Hakkar is pulled he will cast Blood Siphon on the entire raid. In its base state this is a 8 second channeled drain that deals 200 damage per second, stuns, and heals Hakkar for 1000 health per second
		Corrupted Blood: 875-1125 direct shadow damage plus a 200 every 2 seconds shadow DOT that hits a player and several players around.
		Steps to Reproduce
1. Engage Hakkar, watch for DOTS
		Proof
			http://wowwiki.wikia.com/wiki/Hakkar_the_Soulflayer_(tactics)

Edge of Madness
	Not scripted at all

**Comment by Rushor** [comment]
same like the MC issue, i will open up individual tickets for each issue and start working on it tomorrow - ty for the detailed report :)

**Comment by Rushor** [comment]
@chaostheory102: are you shure about that: "Focus Gaze is not LOS’able(line of sight)"

The spell has the attribute AttributesEx2: 0x00000004 (SPELL_ATTR2_CAN_TARGET_NOT_IN_LOS) in dbcs.

could you post some source, that spell http://wotlk.openwow.com/spell=24314 should be LOS’able(?

**Comment by chaostheory102** [comment]
It seems i was mistaken. On our guild site it mentioned the LOS and i
wrote it in. I cannot find a source to say that it is LOS'able.
https://www.youtube.com/watch?v=AWWUfpqpgqk . In that video a paladin seems
to LOS but as well he stops casting, so may be irrelevant.

**Comment by chaostheory102** [comment]
the 2:50 marker

**Comment by Rushor** [comment]
k well then, i will not fix that for now. If you, or one of your guildmember find a source, post it here and i will fix it instantly for PTR :)

**Comment by Rushor** [comment]
alright @chaostheory102 all reported zg issues fixed

**Comment by Klala** [comment]
There is another major bug with Hakkar the Soulflayer that I don't know how you missed but I'll put it here anyway.
Hakkar minds control the tank and he shouldn't. he should mind control anyone BUT the tank.
After the mind control ends there is a problem with moving away from hakkar's general position because since you act as his pet you will move back to his side when you get too far.

https://www.youtube.com/watch?v=LWc2NgfI67E
